requirements_all.txt

```diff
@@ -2408,7 +2408,7 @@ yalesmartalarmclient==0.3.3
 yalexs==1.1.11
 
 # homeassistant.components.yeelight
-yeelight==0.6.3
+yeelight==0.7.0b0
```

Comment:
Would you please ask for a non-beta version to be pushed? https://pypi.org/project/yeelight/